runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -113,6 +113,14 @@
 	private static final String KILLED_TELEBLOCK_OPPONENT_TEXT = "Your Tele Block has been removed because you killed ";
 	private static final String PRAYER_ENHANCE_EXPIRED = "<col=ff0000>Your prayer enhance effect has worn off.</col>";
 	private static final String ENDURANCE_EFFECT_MESSAGE = "Your Ring of endurance doubles the duration of your stamina potion's effect.";
+	private static final String SHADOW_VEIL_MESSAGE = "<col=6800bf>Your thieving abilities have been enhanced.</col>";
```

Comment:
Are these colors the same with transparent chat on/off?

Reply:
Annoyingly they're different, ugh

Reply:
I've fixed this by removing the colour from the start of the messages to check, doing and `endsWith` instead of an `equals`, and a `contains` instead of a `startsWith`. Not sure if there was a better way to do it, but it works.